### ISSUE TITLE
Can't change placeholderColor on MDTextField in Android

### ISSUE THREAD

**Issue by butaminas** [nativescript-community/ui-material-components#191]
### Which platform(s) does your issue occur on?
Android 10
Xiaomi Redmi Note 8
```

CLI: 7.0.8
Cross-platform modules: 7.0.3
Runtime(s): 7.0.0
Plugin(s):
  "dependencies": {
    "@mdi/font": "^5.6.55",
    "@nativescript-community/ui-material-activityindicator": "^5.0.6",
    "@nativescript-community/ui-material-bottomnavigationbar": "^5.0.6",
    "@nativescript-community/ui-material-bottomsheet": "^5.0.6",
    "@nativescript-community/ui-material-button": "^5.0.6",
    "@nativescript-community/ui-material-cardview": "^5.0.6",
    "@nativescript-community/ui-material-core": "^5.0.6",
    "@nativescript-community/ui-material-floatingactionbutton": "^5.0.6",
    "@nativescript-community/ui-material-progress": "^5.0.6",
    "@nativescript-community/ui-material-ripple": "^5.0.6",
    "@nativescript-community/ui-material-slider": "^5.0.6",
    "@nativescript-community/ui-material-tabs": "^5.0.6",
    "@nativescript-community/ui-material-textfield": "^5.0.6",
    "@nativescript-community/ui-material-textview": "^5.0.6",
    "@nativescript/firebase": "11.0.0",
    "@nativescript/core": "7.0.7",
    "@nativescript/theme": "3.0.0",
    "nativescript": "7.0.8",
    "nativescript-vue": "latest",
    "nativescript-vue-navigator": "^1.2.0",
    "vuex": "^3.5.1"
  },
  "devDependencies": {
    "@babel/core": "~7.11.6",
    "@babel/preset-env": "~7.11.5",
    "@nativescript/android": "7.0.0",
    "@nativescript/ios": "7.0.0",
    "@nativescript/webpack": "~3.0.5",
    "babel-loader": "~8.1.0",
    "nativescript-vue-template-compiler": "~2.8.1",
    "node-sass": "^4.7.1",
    "scss-symbols-parser": "^2.0.1",
    "string-replace-loader": "^2.3.0",
    "vue-loader": "~15.9.3",
    "vue-property-decorator": "^9.0.0"
  }
```

Can't seem to be able to change the `placeholderColor` color of `MDTextField`. When I set any value, it seems like the placeholder color is transparent or not visible at all.

Works fine on iOS.

**Comment by farfromrefug** [comment]
@butaminas thanks fixed in 5b0f8ca070501b45e99f581151a2c6394eea33c2

**Comment by farfromrefug** [comment]
@butaminas released 5.0.11

**Comment by butaminas** [comment]
Seems to work fine now.
